script/test-robots

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+"""
+A script to run a test competition match on every robot in an submissions directory.
+"""
+import sys
+import shutil
+import argparse
+import tempfile
+import subprocess
+from typing import List
+from pathlib import Path
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'archives_dir',
+        help=(
+            "The directory containing the teams' robot code, as Zip archives "
+            "named for the teams' TLAs."
+        ),
+        type=Path,
+    )
+    return parser.parse_args()
+
+
+def main(args: argparse.Namespace) -> None:
+    failed_tlas: List[str] = []
+
+    run_comp_match = str(Path(__file__).parent / 'run-comp-match')
+
+    with tempfile.TemporaryDirectory() as tempdir_name:
+
+        tempdir = Path(tempdir_name)
+        # Make a temporary copy of the archives to receive the logs
+        archives = shutil.copytree(args.archives_dir.resolve(), tempdir / 'archives')
+
+        for team in archives.glob('*.zip'):
+            tla = team.stem
+
+            try:
+                print(f"Running match for {tla}")  # noqa:T001
+                subprocess.run(
+                    [
+                        sys.executable,
+                        run_comp_match,
+                        str(archives),
+                        '0',  # no logs will be retained so a fixed match number is used
+                        tla,
+                        '-',
```

Comment:
Suggest using the utils to gather the number of teams and thus make this future proof against the number of corners changing.

Reply:
5371562d6d4883ed0e6f5cc7d2c3d92bfe8a0c07